m/mongodb/Dockerfiles/ubuntu_focal/Dockerfile

```diff
@@ -0,0 +1,130 @@
+# Origin: https://github.com/CrimsonGlory/mongo_ppc64le/blob/master/Dockerfile
```

Comment:
Let's expand this to say that "This file has been adapted from the original work at " https://github.com/CrimsonGlory/mongo_ppc64le/blob/master/Dockerfile" to build on Power, same applies to other files as well, rest looks good

Reply:
Updated, please review